docker/maistra-builder_2.3.Dockerfile

```diff
@@ -76,6 +76,9 @@ RUN curl -sfL https://download.docker.com/linux/centos/docker-ce.repo -o /etc/yu
                    binaryen emsdk docker-ce rubygems npm yarn rpm-build && \
     dnf -y clean all
 
+RUN mkdir /opt/emsdk/node_modules
```

Comment:
Move this to the end of the file with the others mkdirs and chmods

Reply:
ok

Reply:
and use mkdir -p

Reply:
... and use: && \